### PR TITLE
Add metatype properties for SameSite

### PR DESCRIPTION
**Pull request by pnicolucci**
MetaType properties for #10787

### CODE REVIEW

**Comment by pnicolucci** [comment]
@OpenLiberty/message-reviewer I think we are all set here @helyarp  already reviewed these in #10829 I'm just moving them to this PR so I can deliver them without waiting for the rest of the code to be ready so we can meet the translation deadline.

**Comment by pnicolucci** [comment]
#build

**Comment by pnicolucci** [comment]
#build

**Comment by pnicolucci** [comment]
#build

**Comment by pnicolucci** [comment]
@dmuelle  yes these messages SHOULD be different. The work here is for Session Cookies and the other work item is security related cookies.

**Comment by pnicolucci** [comment]
#build

File: dev/com.ibm.ws.session/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -51,6 +51,14 @@ cookie.secure.desc=Specifies that the session cookies include the secure field.
 cookie.http.only=Include HttpOnly field in session cookies
 cookie.http.only.desc=Specifies that session cookies include the HttpOnly field. Browsers that support the HttpOnly field do not enable cookies to be accessed by client-side scripts. Using the HttpOnly field will help prevent cross-site scripting attacks.
 
+# The word SameSite should not be translated
+cookie.same.site=Include SameSite attribute in session cookies
```
Comment (dmuelle) [inline dev/com.ibm.ws.session/resources/OSGI-INF/l10n/metatype.properties]: **change**
Include SameSite attribute in session cookies
**to**
Include a SameSite attribute in session cookies

unless "the" is more appropriate than "a"

**Comment by pnicolucci** [comment]
#build

**Comment by pnicolucci** [comment]
#run-libby-bot

**Comment by skasund** [comment]
L2 message review not required

**Comment by pnicolucci** [comment]
#build

**Comment by pnicolucci** [comment]
#run-libby-bot

**Comment by pnicolucci** [comment]
#run-libby-bot

**Pull request merged**